axum-core/src/extract/default_body_limit.rs

```diff
@@ -72,7 +72,7 @@ use tower_layer::Layer;
 /// [`RequestBodyLimit`]: tower_http::limit::RequestBodyLimit
 /// [`RequestExt::with_limited_body`]: crate::RequestExt::with_limited_body
 /// [`RequestExt::into_limited_body`]: crate::RequestExt::into_limited_body
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Copy)]
```

Comment:
What is `Copy` needed for, in your example? I'm a little bit hesitant about adding it.

Reply:
It allows not using clone when DefaultBodyLimit was constructed during runtime (by reading the config file for example). I understand, that removing it would be a breaking change, but I consider it worth it, since such changes to this layer are unlikely

Reply:
Well, I agree that changes that would make it not `Copy` are somewhat unlikely, but we tend to be conservative about that kind of thing. Anyways, in this specific case I'm fine merging as-is.